examples/tomo/backends/astra_performance_cpu_parallel_2d_cg.py

```diff
@@ -0,0 +1,116 @@
+# Copyright 2014-2016 The ODL development group
+#
+# This file is part of ODL.
+#
+# ODL is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ODL is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ODL.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Performance example of running native astra vs using ODL for reconstruction.
+
+In this example, a 512x512 image is reconstructed using the Conjugate Gradient
+Least Squares method on the CPU.
+
+In general, ASTRA is fater than ODL since it does not need to perform any
```

Comment:
Well here it doesn't use the same things. We both share the projector, but astra then performs arithmetic interally, while we do it externally